### ISSUE TITLE
Allow to load an IDL schema from a location at Neo4j startup

### ISSUE THREAD

**Issue by psypath** [neo4j-graphql/neo4j-graphql#83]
Currently it's possible to push an IDL schema to http://localhost:7474/graphql/idl/.

I would be nice to have a configuration parameter to specify a location from which the plugin will read IDL schema after Neo4j startup. At least, it would simplify usage with Docker.

**Comment by mvid** [comment]
Agreed, scripting around this is frustrating, and doesn't line up with how other Neo4J plugins operate, like ElasticSearch

**Comment by mvid** [comment]
Along with this, it would be great if we could disable the ability to update the schema via client requests. For schemas that don't need to change between deploys, it seems better for security

**Comment by jexp** [comment]
Yes those two are good points
We could limit it to an admin account. So regular users wouldn't be able to do it.
